concordium-consensus/tests/consensus/ConcordiumTests/PassiveFinalization.hs

```diff
@@ -5,394 +5,396 @@
 
 module ConcordiumTests.PassiveFinalization where
 
-import Control.Monad
-import Control.Monad.IO.Class
-import Control.Monad.Trans.State
-import Data.Time.Clock
-import Data.Time.Clock.POSIX
-import qualified Data.Vector as Vec
-import Lens.Micro.Platform
+-- import Control.Monad
```

Comment:
Have you attempted to migrate this? At a glance it does not look so difficult since not that much internal state of basic tree state seems to be used.

Reply:
I have migrated this and updated the PR